uber/site_sections/preregistration.py

```diff
@@ -339,13 +339,12 @@ def process_group_payment(self, session, payment_id, stripeToken):
         else:
             group.amount_paid += charge.dollar_amount
 
-            for attendee in charge.attendees:
+            for attendee in group.attendees:
```

Comment:
Well, that is good to know, but it doesn't quite explain why session.merge _wasn't working_ before. It would clearly find the attendees (because the group's `paid` value would be correct) but the attendees' `paid` property was never updated.